docs/i2s.rst

```diff
@@ -58,6 +68,13 @@ Sets the word clock frequency, but does not start the I2S device if not
 already running.  May be called after ``I2S::begin()`` to change the
 sample rate on-the-fly.
 
+bool setSysClk(int samplerate) 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Changes the PICO system clock to optimise for the desired samplerate. 
+The clock changes to 147.6 MHz for samplerates that are a multiple of 8kHz, and 135.6 MHz for multiples of 44.1kHz.
```

Comment:
The setSysClock actually seems to be for multiples of 8k and 11.025k, not 8k and 44.1k.

Reply:
Imho it would be nice if the documentation mentioned, that this is only necessary when using MCLK. Or does this also have a positive effect when using I2S without MCLK?